test/html/special-cases/fp-191.tpl.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HLS Live Stream with VAST</title>
+    <title>FP-191</title>
```

Comment:
are you sure about this? Leaving the FP ticket number in a file in the repo? If someone clone the FP (public project), it will have that FP-191 and they will not know what it is right?

Reply:
Right! Changed the naming of the files and the README for this. What I think should be expected:
* Internal Tickets, filename: `internal-{short-description}.tpl.html`. Title: `{title}`
* Github issues, filename: `issue-{number}.tpl.html`. Title: `Issue {number} - {title}`